python/tempo/tsdf.py

```diff
@@ -315,6 +316,8 @@ def asofJoin(self, right_tsdf, left_prefix=None, right_prefix="right", tsPartiti
     :param tsPartitionVal - value to break up each partition into time brackets
     :param fraction - overlap fraction
     :param skipNulls - whether to skip nulls when joining in values
+    :param sql_join_opt - if set to True, will use standard Spark SQL join if it is estimated to be efficient
```

Comment:
updated according to master branch - added a doc string for the standard sql join option
There seems to lack a standard of snake_case or camelCase and I followed the latest addition here.